runtime/src/errors/error-codes.js

```diff
@@ -0,0 +1,24 @@
+export const GRAIN_ERR_NOT_NUMBER_COMP = 0;
```

Comment:
(this would be another place to maybe have some OO logic...especially w.r.t. formatting these as error messages)